### PR TITLE
[TS] LPS-102248 | master

### PR DESCRIPTION
**Pull request by sontruongces**
Hi team,

This is the fix for the [LPS-102248](https://issues.liferay.com/browse/LPS-102248) that has passed review at thienquach87#5. Please help me to review and give feedback.

Thanks!

### CODE REVIEW

File: modules/apps/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/util/UADSearchContainerBuilder.java [hunk 1]
```diff
@@ -326,7 +327,15 @@ private SearchContainer<UADEntity<?>> _constructSearchContainer(
 			SearchContainer.DEFAULT_DELTA, currentURL, null,
 			"no-entities-remain-of-this-type", null);
 
-		searchContainer.setId("UADEntities_" + StringUtil.randomId());
+		String uniqueSearchContainerId = (String)renderRequest.getAttribute(
+			"uniqueSearchContainerId");
+
+		uniqueSearchContainerId =
+			Validator.isNotNull(uniqueSearchContainerId) ?
+				uniqueSearchContainerId : StringUtil.randomId();
```
Comment (drewbrokke) [inline modules/apps/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/util/UADSearchContainerBuilder.java:335]: Generally, we avoid ternaries in most application code (but not always). I think we could simplify this like so:

```java
String uniqueSearchContainerId = (String)renderRequest.getAttribute(
	"uniqueSearchContainerId");

if (Validator.isNull(uniqueSearchContainerId)) {
	uniqueSearchContainerId = StringUtil.randomId();
}
```

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Just the one comment. Otherwise this looks good. Thanks!

File: modules/apps/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/util/UADSearchContainerBuilder.java (hunk 1 above)
Comment (sontruongces) [reply modules/apps/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/util/UADSearchContainerBuilder.java]: @drewbrokke I rebased and update the code. Thanks!

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
Looks good, thanks @sontruongces!

**Comment by Sharryxue** [comment]
No related failure in the latest ci:test:relevant result. Will manually forward the commits.

**Comment by Sharryxue** [comment]
Manually forwarded to BChan. https://github.com/brianchandotcom/liferay-portal/pull/123346.
Will closed this pr, correct me if any thing need update. Thanks.

**Comment by Sharryxue** [comment]
ci:close

**Pull request closed without merging**